proto/cosmos/gov/v1/gov.proto

```diff
@@ -130,3 +130,29 @@ message TallyParams {
   //  vetoed. Default value: 1/3.
   string veto_threshold = 3 [(cosmos_proto.scalar) = "cosmos.Dec", (gogoproto.jsontag) = "veto_threshold,omitempty"];
 }
+
+// Params defines the parameters for the x/gov module.
```

Comment:
If we're going to keep the types above, like `VotingParams`, we should mark them deprecated.